core/src/test/scripts/test_data/mutations/data_mutations_annotation_more_than_80_characters.maf

```diff
@@ -0,0 +1,4 @@
+Hugo_Symbol	Entrez_Gene_Id	Center	Tumor_Sample_Barcode	Verification_Status	Validation_Status	Mutation_Status	Sequencer	Chromosome	Start_position	End_position	Variant_Classification	HGVSp_Short	MA:FImpact	MA:link.MSA	MA:link.pdb	SWISSPROT	cbioportal_driver_filter	cbioportal_driver_filter_annotation	cbioportal_multi_filter	cbioportal_multi_filter_annotation
```

Comment:
Minor thing: I would rename the unit test data files to something more descriptive about the column it is testing.